### ISSUE TITLE
Wrap messages the same way as `msgcat`

### ISSUE THREAD

**Issue by mgeisler** [BrettDong/polib#8]
I've noticed that the POT files generated by polib use slightly different wrapping than what `msgcat -o foo.pot foo.pot` gives. I'm tracking this in https://github.com/google/mdbook-i18n-helpers/issues/32, but I would actually suggest improving this here at the source.

**Comment by BrettDong** [comment]
As far as I am aware the text wrapping algorithm is not specified in GNU gettext manual or documentation. Reading GNU gettext source code and then reimplement the text wrapping algorithm in this library is also not an option as it possibly violates the GNU GPL license.

**Comment by mgeisler** [comment]
> As far as I am aware the text wrapping algorithm is not specified in GNU gettext manual or documentation.

I agree, I haven't found anything about this from reading the docs. I recently merged https://github.com/google/mdbook-i18n-helpers/pull/93, which removes part of the churn by wrapping the filenames in the sources field — with some trial and error, I found a setting which seems to be mostly correct.

> Reading GNU gettext source code and then reimplementing the text wrapping algorithm in this library is also not an option, as it may violate the GPL license.

I hadn't thought of that, but it's a good point!

Taking a step back, maybe this isn't really as useful as I first thought. People are generally supposed to use `msgmerge` to update the PO files: this will normalize the wrapping so the polib wrapping of the POT file is not super important.
